src/sentry/models/sentryappinstallation.py

```diff
@@ -25,6 +25,15 @@ class Meta:
         db_table = "sentry_sentryappinstallationtoken"
         unique_together = (("sentry_app_installation", "api_token"),)
 
+    @classmethod
+    def has_organization_access(cls, token, organization):
+        try:
+            install = cls.objects.get(api_token=token).sentry_app_installation
+        except cls.DoesNotExist:
+            return False
+
+        return install.organization == organization
```

Comment:
This looks like it will do 3 queries. Could we eager load any of the associations so django emits fewer queries?

Reply:
@markstory good call, also can just check the organization ids